[2024-07-03T09:57:28.469+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [queued]>
[2024-07-03T09:57:28.478+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [queued]>
[2024-07-03T09:57:28.478+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T09:57:28.490+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 08:55:00+00:00
[2024-07-03T09:57:28.496+0000] {standard_task_runner.py:57} INFO - Started process 20544 to run task
[2024-07-03T09:57:28.502+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T08:55:00+00:00', '--job-id', '352', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp09qjw21q']
[2024-07-03T09:57:28.504+0000] {standard_task_runner.py:85} INFO - Job 352: Subtask id_task_get_emails
[2024-07-03T09:57:28.569+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T09:57:28.651+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T08:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T08:55:00+00:00'
[2024-07-03T09:57:28.652+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T09:57:28.653+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 22, in get_emails
    email_conf = json.load (open(path / ("config/email_conf.json")))
NameError: name 'json' is not defined
[2024-07-03T09:57:28.663+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T085500, start_date=20240703T095728, end_date=20240703T095728
[2024-07-03T09:57:28.674+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 352 for task id_task_get_emails (name 'json' is not defined; 20544)
[2024-07-03T09:57:28.716+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T09:57:28.745+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:04:59.186+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [queued]>
[2024-07-03T12:04:59.198+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [queued]>
[2024-07-03T12:04:59.198+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:04:59.211+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 08:55:00+00:00
[2024-07-03T12:04:59.215+0000] {standard_task_runner.py:57} INFO - Started process 23364 to run task
[2024-07-03T12:04:59.219+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T08:55:00+00:00', '--job-id', '793', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp9myw4_by']
[2024-07-03T12:04:59.220+0000] {standard_task_runner.py:85} INFO - Job 793: Subtask id_task_get_emails
[2024-07-03T12:04:59.272+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:04:59.354+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T08:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T08:55:00+00:00'
[2024-07-03T12:04:59.355+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:04:59.356+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:59.356323_User@StatDep: EmailRobot in screated.
[2024-07-03T12:04:59.356+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:59.356515_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:04:59.356+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:59.356650_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:04:59.731+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:05:00.195+0000] {logging_mixin.py:151} INFO -    mailbox have 1 emails and the robot can process 10
[2024-07-03T12:05:00.602+0000] {logging_mixin.py:151} INFO -    #>2024-07-03 12:05:00.602443_conneciton to db is set.
[2024-07-03T12:05:00.607+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 25, in get_emails
    robot.fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 62, in fetch_emails
    return self.pop3_fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 156, in pop3_fetch_emails
    cursor.execute (query, insert_values)
psycopg2.errors.UndefinedTable: relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^

[2024-07-03T12:05:00.617+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T085500, start_date=20240703T120459, end_date=20240703T120500
[2024-07-03T12:05:00.637+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 793 for task id_task_get_emails (relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^
; 23364)
[2024-07-03T12:05:00.649+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T12:05:00.674+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:23:06.872+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [queued]>
[2024-07-03T12:23:06.891+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [queued]>
[2024-07-03T12:23:06.891+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:23:06.906+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 08:55:00+00:00
[2024-07-03T12:23:06.915+0000] {standard_task_runner.py:57} INFO - Started process 24234 to run task
[2024-07-03T12:23:06.922+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T08:55:00+00:00', '--job-id', '1094', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpvq7ms0pr']
[2024-07-03T12:23:06.923+0000] {standard_task_runner.py:85} INFO - Job 1094: Subtask id_task_get_emails
[2024-07-03T12:23:06.977+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T08:55:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:23:07.049+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T08:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T08:55:00+00:00'
[2024-07-03T12:23:07.050+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:23:07.051+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:23:07.051180_User@StatDep: EmailRobot in screated.
[2024-07-03T12:23:07.051+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:23:07.051361_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:23:07.051+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:23:07.051505_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:23:07.447+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:23:07.727+0000] {logging_mixin.py:151} INFO -    mailbox have 0 emails and the robot can process 10
[2024-07-03T12:23:07.728+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:23:07.727919_User@StatDep: no emails to process, exiting pop3_fetch_emails()...
[2024-07-03T12:23:07.728+0000] {python.py:194} INFO - Done. Returned value was: True
[2024-07-03T12:23:07.774+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T085500, start_date=20240703T122306, end_date=20240703T122307
[2024-07-03T12:23:07.818+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-03T12:23:07.850+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
